### ISSUE TITLE
Is English documentation available somewhere?

### ISSUE THREAD

**Issue by ComFreek** [gfngfn/SATySFi#235]
So far I could only locate Japenese documentation in this repo's demo files and wiki.

Is English documentation available somewhere?

**Comment by nekketsuuu** [comment]
Currently, there is no English documentation which fully describes SATySFi's features. AFAIK, it's a TODO task.

**Comment by gfngfn** [comment]
Thank you for having interest for this project.

Although I have long been willing to write some book for introducing SATySFi in English, we still haven’t got around to it yet as @nekketsuuu -san mentioned.

I have already provided a Japanese version, [*The SATySFi​book*](https://gfngfn.booth.pm/items/1127224), which has 160 pages and had taken approximately one month of full-time work on writing. It thereby probably takes at least half an year to write English version by myself… It may be worth considering to recruit some paid contributors for translation.

**Comment by ComFreek** [comment]
For me just having an English extended abstract (1 - 2 pages) describing what this project is about and how exactly it solves things would be enough as a starter.

I am primarily interested in this project from a language design perspective. Currently, I am collecting some design ideas for a TeX successor and certainly "statically typed" is on my list. And this project claims to be statically typed :smile:

**Comment by claudio-naoto** [comment]
Hi, I'm interested in this project and made a translation of the demo file to english and brazilian portuguese (mainly using google translator). It is not perfect but maybe it is useful. I can send the files if someone find this interesting.

**Comment by ComFreek** [comment]
@claudio-naoto Yes, please! Thanks for your initiative. Can you publicly share them somewhere, e.g. on GitHub Gists?

**Comment by claudio-naoto** [comment]
Ok. 
[English demo](https://gist.github.com/claudio-naoto/419285013ee08714b0f6c3db62b02c91)
[Brazilian portuguese demo](https://gist.github.com/claudio-naoto/d7d339879b2444b02d9ce35fdd9595a1)

**Comment by claudio-naoto** [comment]
Do you know where/how I should change environment labels like "mokuji" to "Table of Contents"?

**Comment by na4zagin3** [comment]
@claudio-naoto Thank you so much!

> Do you know where/how I should change environment labels like "mokuji" to "Table of Contents"?

It's hard-coded in the class package `stdjabook`.

https://github.com/gfngfn/SATySFi/blob/360d9417b9ee26fcc89016b9d42c5b27c8b1e304/lib-satysfi/dist/packages/stdjabook.satyh#L393

AFAIK, as of now, there's no class packages with a configurable TOC section header string.

I'm now developing a class file for Latin-script languages, but not yet ready..

https://github.com/na4zagin3/satysfi-fss/blob/master/src/class-nzla/nzla.satyh

**Comment by na4zagin3** [comment]
The quickest hack is to copy `stdjabook.saty` to the same directory of the document, e.g., `./stdbook.saty`, replace `{目次}` with `{Index}`, `{Índice}`, or anything, and replace the import directive `@require stdjabook` with `@import ./stdbook`.
